TESTING.md

```diff
@@ -9,23 +10,48 @@ The integration test cases can be executed by using the `Makefile` recipe or fro
 		StorageClass: "replication-unity",
```

Comment:
Can we replace unity with one of supported platform?

Reply:
and done.